neuro_san/service/grpc/grpc_agent_server.py

```diff
@@ -53,7 +52,7 @@ class GrpcAgentServer(AgentServer, AgentStateListener):
     def __init__(self, port: int,
                  server_loop_callbacks: ServerLoopCallbacks,
                  network_storage_dict: Dict[str, AgentNetworkStorage],
-                 agent_networks: Dict[str, AgentNetwork],
+                 server_status: ServerStatus,
                  server_name: str = DEFAULT_SERVER_NAME,
```

Comment:
We need server_status, we don't need agent_networks anymore - they are used outside for initial network_storage setup.